fecgov/openFEC
# Add is_active column into committee/<committee_id>/history/

## Summary (required)
To make it more clear to our users what the status of a committee is, we want to add an indicator at the top of the committee profile page.

- Resolves #[_4284_]

_Include a summary of proposed changes._
model: CommitteeHistory
test_committees.py

## How to test the changes locally
1)checkout branch
2)pytest
3)test url to see new field: is_avtive
on local:
a) is_active=true
http://127.0.0.1:5000/v1/committee/C00693234/history

b) is_active=false:
http://127.0.0.1:5000/v1/committee/C00000141/history/
